devops/git/workflow.md

```diff
@@ -11,7 +11,8 @@ after the initial commit, with the correct protection rules in place.
 * `feature/*` branches created from master for each feature
 * `fix/*` branches are for non-urgent fixes, and follow the same flow as feature branches.
 * `hotfix/*` branches should only be created off of `staging` or `production` and merged accordingly for urgent fixes only. However, they can be [cherry picked](https://git-scm.com/docs/git-cherry-pick) into other branches if absolutely necessary.
-* `staging` should contain the latest, customer visible changes and be on the `Workshop`
+* `qa` should contain the latest, quality assurance (internal testing) visible changes and be on the `Shed` (`qa.project-name.langleyfoxall.app`)
```

Comment:
Thoughts on standardizing capital letters in branch names? I know some repositories have capital letters for `UAT`.

Reply:
Personally, I prefer always having lowercase branch names. I believe it is a standard of sorts - I can't say I've ever seen a branch name with capitals in the wild.

Reply:
> I believe it is a standard of sorts - I can't say I've ever seen a branch name with capitals in the wild.

But as I've seen it in repositories internally it may be worth adding a line describing it though.

Reply:
I don't think it matters tbh, as long as it's standard across a project I don't it's something we should care about.

Reply:
I kind of in agreement with @AlexCatch here. Most branches are ephemeral so their names are pretty inconsequential. As long as the core branch names remain consistent with what is defined here, I don't see much of a problem.

However, I do prefer lower-case kebab-cased branch names, so it would not hurt to add in to a separate PR.

Reply:
Fair enough 🤷‍♂ it's just worth noting that branches are case sensitive so if someone is setting up on the Shed then it might cause issues.